osa_cli_releases/releasing.py

```diff
@@ -251,6 +251,8 @@ def update_ansible_role_requirements_file(
 
         copyreleasenotes = False
 
```

Comment:
oh wow, this code smells. Sorry for that, I see that I wrote it from a bash conversion _really quickly_